src/core/src/op/convert.cpp

```diff
@@ -57,7 +57,7 @@ struct Evaluate : public element::NoAction<bool> {
                           CONVERT_ET_LIST,
                           EvalByOutputType,
                           out.get_element_type(),
-                          iterator<ET_IN>(arg.data()),
+                          iterator<ET_IN>(reinterpret_cast<const TI*>(arg.data())),
```

Comment:
Is root issue caused that `data()` is not return `const void*`, which cause wrong selection of overloaded convert function?

Reply:
Right

Reply:
Ok thanks for update, it should be resolved in 2026.0 as this member will return const void* if tensor is constant.